app/src/main/java/com/hedvig/app/feature/offer/ui/OfferAdapter.kt

```diff
@@ -57,7 +60,7 @@ class OfferAdapter(
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        holder.bind(getItem(position), fragmentManager, tracker, removeDiscount, marketManager)
+        holder.bind(getItem(position), fragmentManager, tracker, onRemoveDiscount, onSign, marketManager)
```

Comment:
Perhaps we could provide `onSign` in the `ViewHolder`-`constructor` instead? You've introduced that pattern before and I think it makes the `ViewHolder`-binding process a whole lot cleaner 🙂